guides/mapping-and-blending/avoiding-css-conflicts/README.md

```diff
@@ -1,39 +1,20 @@
 # Avoiding CSS conflicts
 
-Complex web apps are composed of HTML responses from multiple apps. This means that the merged HTML, CSS and JavaScript runs in the global namespace.
+When your system is composed of HTML responses from multiple apps, there's always a risk of naming conflicts and cascading side effects in your CSS. A solution to this is to use a convention that isolates styles coming from different apps.
 
-With a global namespace, there's always the risk of CSS conflicts and leaks that cause side effects in other apps. We recommend using the Block Element Modifier (BEM) methodology to avoid these side effects.
+We recommend BEM, a well-known convention that solves this problem.
 
-## BEM
+## What is BEM?
 
-Due to limited browser support for technical solutions to CSS conflicts, the primary way right now to avoid CSS conflicts is to use a naming convention. BEM is one of these naming conventions. 
+In BEM naming convention, you **only use classes** in your stylesheets.  There are three kinds of classes at your disposal: blocks, elements, and modifiers.
 
-The basic principle in BEM is to **only use classes in your stylesheets**.
+A **block** is a basic class that represents a logical area of your app's UI (*a menu, login form, a search form*).
 
-These classes should be named as combinations of Blocks, Elements, and Modifiers. 
+An **element** is a smaller fragment of a block that performs a particular function (*a link in the menu, a password input in the login form, a search icon*). The element part of a class starts after `__`.
```

Comment:
Fixed. I put **smaller** for emphasis.

I will put the part about `__` and `--` inside the CSS snippet.